### ISSUE TITLE
Other is not cleared when a selection is changed

### ISSUE THREAD

**Issue by danielruss** [episphere/quest#376]
if you select other, then go back and select another response. The other value is not clear.

This is the original #347 

```
[DIS] what disease

(1) dis 1
(2) dis 2
(3) other |__|id=other|

[WEA] what whether
[1] good
[2] bad
[3] other |__|id=other|


[END]
{$DIS.DIS}
{$disOther}
{$WEA}
{$weaOther}
```

**Comment by danielruss** [comment]
@Davinkjohnson   Just pushed a fix.  Please let the questionnaire developers know.  Major code change. Much easier to understand and better documented.

**Comment by danielruss** [comment]
This may fix some of the other buttons.  But if you deselect an other, the text box gets cleared.

**Comment by boyd-mj** [comment]
@joshid-ims can you and/or Gwen see if this addresses the issues we were seeing that I reported in issue #349

**Comment by danielruss** [comment]
#349 is a pull request to fix one of the many issues discussed in #347.

**Comment by joshid-ims** [comment]
Gwen is testing this.
